testsuite/MDAnalysisTests/transformations/test_nojump.py

```diff
@@ -0,0 +1,57 @@
+###
+
+import numpy as np
+import pytest
+from numpy.testing import assert_array_almost_equal
```

Comment:
We use [`assert_allclose`](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html) nowadays but haven't updated everything.